httpd/handler.go

```diff
@@ -667,6 +673,99 @@ func (h *Handler) serveProcessContinuousQueries(w http.ResponseWriter, r *http.R
 	w.WriteHeader(http.StatusAccepted)
 }
 
+func (h *Handler) serveRunMapper(w http.ResponseWriter, r *http.Request) {
+	// we always return a 200, even if there's an error because we always include an error object
+	// that can be passed on
+	w.Header().Add("content-type", "application/json")
+	w.WriteHeader(200)
```

Comment:
You don't need to do this, since it happens automatically right? Perhaps you just want to be explicit?